src/sagemaker/session.py

```diff
@@ -30,6 +30,7 @@
 from sagemaker import vpc_utils
 from sagemaker.user_agent import prepend_user_agent
 from sagemaker.utils import name_from_image, secondary_training_status_changed, secondary_training_status_message
+from sagemaker.exceptions import UnexpectedStatusException
```

Comment:
Let's try to follow Google Python Style guide, http://google.github.io/styleguide/pyguide.html#224-decision

```python
from sagemaker import exceptions
```

Reply:
updated.
I have to mention that other imports in the same file are quite far from the guide